rust/rope/src/delta.rs

```diff
@@ -16,6 +16,8 @@
 //! It's useful to explicitly represent these operations so they can be
 //! shared across multiple subsystems.
 
+#![allow(clippy::many_single_char_names)]
```

Comment:
I would prefer this to be on a specific function of that is possible?

Reply:
Fixed.